index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <h1 class="header-title">My Logo</h1>
+    <i class="fa fa-bars fa-lg bars"></i>
+  </header>
```

Comment:
- [ ] [REQUIRED] Good job so far! The `hamburger` button should have a `nav` tag. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices 👍🏼